src/hipscat_import/catalog/arguments.py

```diff
@@ -58,156 +63,78 @@ class ImportArguments:
         dask_threads_per_worker (int): number of threads per dask worker.
     """
 
-    def __init__(
-        self,
-        catalog_name="",
-        epoch="J2000",
-        catalog_type="object",
-        input_path="",
-        input_format="parquet",
-        input_file_list=None,
-        ra_column="ra",
-        dec_column="dec",
-        id_column="id",
-        add_hipscat_index=True,
-        use_schema_file=None,
-        output_path="",
-        overwrite=False,
-        resume=False,
-        highest_healpix_order=10,
-        pixel_threshold=1_000_000,
-        debug_stats_only=False,
-        filter_function=None,
-        file_reader=None,
-        tmp_dir="",
-        progress_bar=True,
-        dask_tmp="",
-        dask_n_workers=1,
-        dask_threads_per_worker=1,
-    ):
-        self.catalog_name = catalog_name
-        self.epoch = epoch
-        self.catalog_type = catalog_type
-        self._input_path = file_io.get_file_pointer_from_path(input_path)
-        self.input_format = input_format
-        self._input_file_list = (
-            [file_io.get_file_pointer_from_path(x) for x in input_file_list]
-            if input_file_list
+    epoch: str = "J2000"
+    catalog_type: str = "object"
+    input_path: str = ""
```

Comment:
Could this type be `FilePointer` if we're using it as one and it's being set to one in the post_init?